index.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Persaonal Portfolio</title>
+    <link rel="stylesheet" href="styles.css" />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.2.1/css/all.min.css"
+      integrity="sha512-MV7K8+y+gLIBoVD59lQIYicR65iaqukzvf/nwasF0nqhPay5w/9lJmVM2hMDcnK1OnMGCdVK+iQrJ7lzPJQd1w=="
+      crossorigin="anonymous"
+      referrerpolicy="no-referrer"
+    />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Inter:wght@600&display=swap"
+      rel="stylesheet"
+    />
+  </head>
+  <body>
+    <nav>
+      <ul>
+        <li><h1 class="welcome">Welcome</h1></li>
+
+        <li><i class="fa-solid fa-bars fa-2x"></i></li>
+      </ul>
+    </nav>
+    <header class="header-section">
+      <h2 class="header-section-title">
+        <p class="header-name">Hey there. I’m Coral</p>
+        <p class="header-name header-title">I’m a software developer</p>
+      </h2>
+    </header>
+    <p class="paragraph">
+      I can help you build a product , feature or website Look through some of
+      my work and experience! If you like what you see and have a project you
+      need coded, don’t hestiate to contact me.
+    </p>
+    <div class="wrapped">
+      <ul>
+        <li><i class="fa-brands fa-twitter"></i></li>
+        <li><i class="fa-brands fa-linkedin"></i></li>
+        <li><i class="fa-brands fa-github"></i></li>
+        <li><i class="fa-brands fa-angellist"></i></li>
+      </ul>
```

Comment:
- [ ] Good job using the ul and li tags :clap: , I think the div above it is not necessary. 
- [ ] I see you are having trouble resizing and placing the icons as on the Figma design, I suggest doing the following:

  - Export icons from the Figma design as PNG or SVG
  - put them in the li :`<li><a  href="#"><img src="linkedIn.png"></a<li/>`
  - then considering the ul tag that contains all the li, is a container you can resize it the way you want to put the icons in the middle as on the Figma design (using the right margins and paddings) :point_down: 

your design:
![Screenshot from 2023-01-19 13-32-29](https://user-images.githubusercontent.com/97834160/213431930-697822ea-637b-461d-a87a-b8e195c229f2.png)



Figma  design:

![Screenshot from 2023-01-19 13-30-49](https://user-images.githubusercontent.com/97834160/213431617-22b7915b-1ee4-4802-ab41-88820f20688e.png)